cmake/test-targets.cmake

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.3.0)
+
+enable_testing()
+
+# add test for install target
```

Comment:
these comments sound a bit like left-over notes from your TODO-list? :)

Reply:
Do you want me to remove the comment because you find it useless? I like it even just for the sake of consistency (every paragraph has a comment).

Reply:
If you like to keep it then please rephrase to not be imperative, e.g. just "test for install target"